### DISCUSSION TITLE
Blog 2: Scrum und UI

### DISCUSSION THREAD

**Discussion by Max-Rohrhirsch** [DHBW-UMOC/UMOC#6] [Announcements]
## Scrum
Wegen Mängeln an Features in GitHub Projects sind wir auf Jira umgestiegen.

### UI
Für den Chat haben wir folgendes Design entworfen.
Uns fehl allerdings noch, eine Navigationsleiste in der man neue Kontakte und Gruppen hinzufügen kann.
![UMOC](https://github.com/user-attachments/assets/879b9ed5-3aef-4aad-a6aa-af0170aa2f76)

### UML
![UMOC_Anwendungsfalldiagramm](https://github.com/user-attachments/assets/51169c5c-363b-460f-9887-2966c566d4da)

**Comment by Kartoffeleintopf** [comment]
Euer Blog hat ansichtlich dargestellt wie ihr euch euren Messenger vorstellt. Interressieren würde mich besonders die "UMOC Features", welche in eurem UML aufgezeigt werden. Das UI des Chats sieht auch nicht schlecht aus, und ihr habt sogar ein nettes Logo. Als Nutzer von GitHub Projects würde mir eine Erläuterung euer Gründe für den Wechsel zu Jira sehr zusagen.

> Euer TripAhead Team (CN)

**Comment by OrlandoLegolas** [comment]
Euer Blog gibt einen klaren Einblick in eure Vision für den Messenger! Besonders interessant finden wir die „UMOC Features“, die ihr im UML-Diagramm dargestellt habt – viele der Ideen heben sich deutlich von den üblichen Messenger-Funktionen ab und klingen sehr innovativ, wenn auch wahrscheinlich sehr kontrovers. Auch das UI-Design des Chats sieht vielversprechend aus, und das Logo ist ebenfalls gut gelungen.

Wir würden gerne mehr über eure Entscheidung erfahren, von GitHub Projects zu Jira zu wechseln. Es wäre spannend zu verstehen, welche Features euch bei GitHub gefehlt haben und warum Jira für euch besser geeignet ist. Zudem finden wir es gut, dass ihr das UI weiterentwickelt, insbesondere die geplante Navigationsleiste zum Hinzufügen von Kontakten und Gruppen scheint eine wichtige Ergänzung für die Benutzerfreundlichkeit zu sein!

Wir feuen uns immer von euch zu lesen.

Beste Grüße 
Euer GymTrackerX Team
